app/models/feedback.rb

```diff
@@ -52,20 +53,23 @@ def description
 
   def save_feedback
     response = SurveyMonkeySender.call(self)
-    @response_failed_message = "Unable to submit feedback [#{response[:error_code]}]" unless response[:success]
+    if response[:success]
+      @response_succeeded_message = 'Feedback submitted'
+    else
+      @response_failed_message = "Unable to submit feedback [#{response[:error_code]}]"
+    end
```

Comment:
> Only one of @response_succeeded_message and @response_failed_message is used so maybe there only needs to be one @response_message.

👍  good shout!

Use of the  ternary was considered and decided against as it would 
>...be clearer still to have it over multiple lines...